common/lib/capa/capa/templates/jstextline.html

```diff
@@ -1,3 +1,4 @@
+<%! from openedx.core.djangolib.markup import HTML %>
```

Comment:
This file basically hasn't been touched since it was written by Ike in May, 2012. I can't find any references to it in the codebase, nor in documentation.

@sstack22 OK to delete?

FYI @muzaffaryousaf I didn't see it in your document either.